user-org-service/app/controllers/logsmanager/validator/LogValidator.java

```diff
@@ -0,0 +1,52 @@
+package controllers.logsmanager.validator;
+
+import java.util.*;
+import org.apache.commons.lang3.EnumUtils;
+import org.sunbird.util.LoggerEnum;
+import org.sunbird.util.ProjectLogger;
+import org.sunbird.util.UserOrgJsonKey;
+import org.sunbird.util.request.Request;
+
+public class LogValidator {
+
+  /**
+   * This method will valid the passed level is supported by service or not
+   *
+   * @param request
+   * @return boolean
+   */
+  public static boolean checkLogValidationError(Request request) throws Exception {
```

Comment:
Split this taks in two method.
1. isRequestPresent
2. isLogLevelValid
both method should not throw any exception.